<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgNuevaPersona</class>
 <widget class="QDialog" name="dlgNuevaPersona">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="txtNombre">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtApellidos">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>58</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>58</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Apellidos</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>561</width>
     <height>291</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tabGeneral">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QCheckBox" name="ckBuscado">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>84</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscado</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckWiki">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>84</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Wikipedia</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtWiki">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>341</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckViaf">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>84</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Viaf</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtViaf">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>341</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabNotas">
    <attribute name="title">
     <string>Notas</string>
    </attribute>
    <widget class="QTextEdit" name="txtNotas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>531</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabOtros">
    <attribute name="title">
     <string>Otros datos</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabOtrosNombres">
    <attribute name="title">
     <string>Otros nombres</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabReferencias">
    <attribute name="title">
     <string>Referencias</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QLineEdit" name="txtNacimiento">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMuerte">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>58</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Nacimineto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>58</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Muerte</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btCancelar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>410</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btOK">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>410</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
